css/index.css

```diff
@@ -1 +1,352 @@
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: column;
+    border: none;
+    border: 1px solid blue;
+  }
+}
+nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  height: 50px;
+}
+nav a {
+  text-decoration: none;
+  color: black;
+  font-size: 1.5rem;
+}
+nav a:hover {
+  font-weight: 800;
+}
+@media (max-width: 500px) {
+  nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+    border: 0;
+  }
+  nav a {
+    border-bottom: 1px solid #A96A67;
+    padding: 1%;
+    width: 100%;
+    text-align: center;
+  }
+  nav a:first-child {
+    border-top: 1px solid #A96A67;
+  }
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  background-color: #ceaaa8;
+  padding: 5%;
+  border-bottom: 5px solid black;
+}
+header img {
+  height: 20%;
+  width: 20%;
+  border-radius: 100px;
+}
+header .intro {
+  width: 75%;
+}
+@media (max-width: 500px) {
+  header {
+    display: flex;
+    justify-content: center;
+    align-items: space-around;
+    flex-direction: column;
+    text-align: center;
+  }
+  header img {
+    display: none;
+  }
+}
+.middle-content {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  padding: 1% 0;
+  height: 70vh;
+}
+.middle-content .middle-left-content {
+  height: 99%;
+  width: 30%;
+  padding: 0 2%;
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex-direction: column;
+}
+.middle-content .middle-left-content li {
+  list-style: none;
+  padding: 5px 0;
+}
+.middle-content .middle-left-content .skillsbox {
+  padding: 5% 5% 5% 0;
+}
+.middle-content .middle-left-content .contact {
+  padding: 5% 5% 5% 0;
+}
+.middle-content .middle-left-content .contact p {
+  padding: 5% 0;
+}
+.middle-content .workhistory {
+  height: 99%;
+  width: 70%;
+  padding: 0 5%;
+  border-left: dashed 2px #A96A67;
+}
+.middle-content .workhistory .job {
+  padding: 1%;
+  padding-bottom: 5%;
+}
+.middle-content .workhistory .job .title {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  flex-direction: row;
+  padding-bottom: 1%;
+}
+.middle-content .workhistory .job ul {
+  padding-top: 1%;
+}
+.middle-content .workhistory .education .school {
+  padding: 1% 0;
+}
+.middle-content .workhistory .education .title {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  flex-direction: row;
+  padding-bottom: 1%;
+}
+@media (max-width: 500px) {
+  .middle-content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: column;
+  }
+  .middle-content .middle-left-content {
+    width: 100%;
+    border: none;
+    padding: 5%;
+  }
+  .middle-content .middle-left-content .skillsbox {
```

Comment:
if you do a display : flex on your ul tags, and then do a justify-content : space-between. that will format your skills nicely for mobile.  do all of this in your @media (max-width : 500px) media query.  

Also, for mobile, set your flex-direction to column on the .skills-box, that will keep your h3 tags above the skills.  play around with the justify content properties, and play around with it.  This is some general guidance for formatting your text in mobile.